shared/build.gradle.kts

```diff
@@ -60,11 +62,7 @@ kotlin {
         val iosX64Main by getting
         val iosArm64Main by getting
         val iosSimulatorArm64Main by getting
```

Comment:
we don't need it anymore

Reply:
Can you be more specific? Do you mean iosX64Main -> iosSimulatorArm64Main?

Reply:
I mean, after removing the manual setup of the iosMain source set, we don't need these properties anymore